packages/issuer-rest/lib/expressUtils.ts

```diff
@@ -0,0 +1,21 @@
+import { Request } from 'express'
+
+export const validateRequestBody = ({
```

Comment:
Either make the name more clear, or just throw an error from here (and catch it).

Reply:
Fixed